Apps/WebClient/src/ClientApp/src/services/restCredentialService.ts

```diff
@@ -1,10 +1,7 @@
 import { injectable } from "inversify";
 
 import { ExternalConfiguration } from "@/models/configData";
-import {
-    CredentialConnection,
-    WalletCredential,
-} from "@/models/credentialConnection";
+import { WalletConnection, WalletCredential } from "@/models/credential";
```

Comment:
can we change the name from credential to wallet?!

Reply:
IMO wallet is an implementation detail for the credential types

Reply:
Renamed the file to `wallet.ts` per discussion.